main/resource-access.bs

```diff
@@ -130,6 +130,88 @@ Note:
 Servers that wish to disable URI re-use may want to use the `410` status
 code.
 
+### Resource type heuristics ### {#resource-type-heuristics}
+
+Servers MUST apply the following heuristics to determine a request's resource
+type
+[[Source](https://github.com/solid/specification/issues/128#issuecomment-573033297)].
+
+<table id="heuristics-resource-type">
+  <caption>Heuristics to determine a resource type</caption>
+  <thead>
+    <tr>
+      <th>`Slug`</th>
```

Comment:
Can we also include Resource Name (in addition to a Slug) in that column, for creating resources via PUT?